backend/capellacollab/tools/models.py

```diff
@@ -40,6 +40,71 @@ class ToolIntegrations(pydantic.BaseModel):
     )
 
 
+RESOURCES_DOCS = (
+    "To find the right value, you can use the Kubernetes dashboard to monitor the resource usage of the tool. "
+    "Refer to the Kubernetes documentation for more details: "
+    "https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits"
+)
+
+
+class CPUResources(pydantic.BaseModel):
+    requests: float = pydantic.Field(
+        default=0.4,
+        description=(
+            "Each session gets at least the specified amount of physical or virtual CPU cores. "
+            "An inaccurate value can lead to higher costs. " + RESOURCES_DOCS
+        ),
+        le=4,
+        gt=0,
+        examples=[0.5, 2],
+    )
+    limits: float = pydantic.Field(
+        default=2,
+        description=(
+            "Each session can not consume more than the specified amount of physical or virtual CPU cores. "
+            "A high value can lead to high costs, while a low value can lead to performance issues. "
+            + RESOURCES_DOCS
+        ),
+        le=8,
+        gt=0,
+        examples=[0.8, 3],
+    )
+
+
+class MemoryResources(pydantic.BaseModel):
+    requests: str = pydantic.Field(
+        default="1.6Gi",
+        description=(
+            "Each session gets at least the specified amount of memory. "
+            "An inaccurate value can lead to higher costs. " + RESOURCES_DOCS
+        ),
+        examples=["100Mi", "1.6Gi", "2Gi"],
+    )
+    limits: str = pydantic.Field(
+        default="6Gi",
```

Comment:
Seeing mixed numbers and measures makes me nervous but if these are the expected values for configurations I guess it would just be extra work and no gain to store them as enumerated strings or as a combination number + string measurement?

Reply:
Why does that make you nervous? I would actually say that it makes the documentation easier to read. Split entries are harder to read.

Reply:
I'm not _actually_ worried about it, but they can't be validated or constrained as easily (e.g. 500Gi instead of Mi, text other than Mi or Gi)

Reply:
Validation for valid values should be simple (a simple regex which checks for number + valid unit). 
Enforcing an upper bound is harder, we would need to resolve the unit for it (but this would be the same with separate fields for number + unit). Just a single number is less readable, I'd say.